src/test/ui/optimization-remark.rs

```diff
@@ -0,0 +1,19 @@
+// build-pass
+// ignore-pass
+// no-system-llvm
```

Comment:
Please specify a minimum LLVM version here instead (13.0.0 if no more specific known) -- at least it doesn't look like this depends on any modifications in our fork?

Reply:
My only concern here was that remark message is not stable enough to match it across different LLVM versions. For example, that seems to be a recent addition of a quotes around function names in InlineAdvisor.

Reply:
Hm, okay, let's go with this for now then.